wikimedia/wikipedia-ios
# add WMF_ANNOUNCEMENT_DATE_IGNORE macro for testing announcements

Testing steps:

1. Switch to Beta scheme.
2. Fresh launch app.
3. Background then foreground.
4. Pull to refresh on Explore feed.
5. Go to an article

Confirm you see announcement modal on article. Do the same steps for Wikipedia scheme and confirm you do not see announcement modal on article.
